angular.js

```diff
@@ -4,5 +4,59 @@ module.exports = {
         'plugin:@angular-eslint/recommended--extra',
         'plugin:@angular-eslint/template/process-inline-templates',
         './typescript'
-    ]
+    ],
+    rules: {
+        /*
+            Overrides to Angular recommended rules:
+            https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.json
+        */
+
+        '@angular-eslint/component-class-suffix': 'error',
+        '@angular-eslint/contextual-lifecycle': 'error',
+        '@angular-eslint/directive-class-suffix': 'error',
+        '@angular-eslint/no-conflicting-lifecycle': 'error',
+        '@angular-eslint/no-empty-lifecycle-method': 'error',
+        '@angular-eslint/no-host-metadata-property': 'error',
+        '@angular-eslint/no-input-rename': 'error',
+        '@angular-eslint/no-inputs-metadata-property': 'error',
+        '@angular-eslint/no-output-native': 'error',
+        '@angular-eslint/no-output-on-prefix': 'error',
+        '@angular-eslint/no-output-rename': 'error',
```

Comment:
We discussed this in person. General consensus is that we can probably disable this rule because changing input names is a deliberate act which is almost impossible to do accidentally. @TrevorKarjanis will try out this rule in a codebase and create a proposal for voting.

Reply:
In testing, exceptions to this rule are common when using the $ postfix.

**Proposal**
```javascript
/*
    Disallow renaming directive outputs generally. Use an inline comment to disable the rule for exceptions where the 
    context changes significantly, or disable the rule for projects that use the dollar sign postfix for EventEmitters.
*/
'@angular-eslint/no-output-rename': 'error'
```

Reply:
We have decided to accept this proposal.